docs/restrict-cross-namespace-associations.asciidoc

```diff
@@ -6,22 +6,22 @@ endif::[]
 [id="{p}-restrict-cross-namespace-associations"]
 === Restrict cross namespace resource associations
 
-By default, associations can be created between any resources as long as they are deployed in some namespaces managed by the same ECK instance.
+By default, associations can be created between any resources as long as they are deployed in namespaces managed by the same ECK instance.
 
-This is true, for example, when using the `elasticsearchRef` field in order to automatically configure and establish a trust relationship between Kibana and Elasticsearch. If a user can create a Kibana instance in a namespace managed by ECK then this Kibana instance can be associated with any Elasticsearch cluster deployed in another namespace managed by the same ECK instance.
+This is true, for example, when using the `elasticsearchRef` field to automatically configure and establish a trust relationship between Kibana and Elasticsearch. If a user can create a Kibana instance in a namespace managed by ECK, then this Kibana instance can be associated with any Elasticsearch cluster deployed in another namespace managed by the same ECK instance. Even if the user only has permissions to edit Kibana and not the Elasticsearch resource.
 
 This section describes how you can restrict the associations that can be created between resources managed by ECK.
 
 [float]
 ==== Enabling cross namespace association restriction
 
-This feature is disabled by default. To enable it start the operator with the `--enforce-rbac-on-refs` flag.
+This feature is disabled by default. To enable it, start the operator with the `--enforce-rbac-on-refs` flag.
 
 NOTE: This feature only provides an access control for resources deployed across two different namespaces. Associations can still be freely created between resources deployed in a same namespace.
 
-Once enabled, associations are allowed to be created as long as the `ServiceAccount` specified in the associated resources can execute a `get` request against the referenced Elasticsearch object.
+Once enabled, associations can be created as long as the `ServiceAccount` specified in the associated resources can execute a `get` request against the referenced Elasticsearch object.
```

Comment:
maybe "HTTP GET request" or if you want to phrase it in terms of Kubernetes RBAC: "`get` resource request" ? Not sure myself, was just confused by the lowercase `get` I guess...